Orange/widgets/regression/owsgdregression.py

```diff
@@ -180,6 +186,15 @@ def _on_func_changed(self):
         for spin, enabled in zip(self._func_params, mask):
             spin.setEnabled(enabled)
 
+    def send_report(self):
+        items = OrderedDict()
+        items['Loss Function'] = self.loss_function_options[self.loss_function].capitalize()
+        items['Penalty'] = self.penalty_options[self.penalty_type].capitalize()
+        items['Learning Rate'] = self.learning_rate_options[self.learning_rate].capitalize()
+        items['No of Iterations'] = self.n_iter
+        self.report_items("Model parameters", items)
```

Comment:
Benevolent dictator of reports (= me) prefers: "Loss function", "Learning rate", "Number of iterations". Sentence case, no unnecessary abbr. :)